front/src/common/Map/Settings/MapSettingsSignals.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC, useCallback } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { fromPairs } from 'lodash';
+
+import SwitchSNCF from 'common/BootstrapSNCF/SwitchSNCF/SwitchSNCF';
+import stopsIcon from 'assets/pictures/layersicons/layer_stops.svg';
+import lightsIcon from 'assets/pictures/layersicons/layer_signal.svg';
+import { updateSignalsSettings } from 'reducers/map';
+import { RootState } from 'reducers';
+
+type SettingsSignals = RootState['map']['signalsSettings'];
+const CONSTS_SETTINGS: Array<keyof SettingsSignals> = ['all', 'stops', 'lights'];
```

Comment:
I don't understand this note
Why adding this Partial here ?

Reply:
I'm not a big fan of the name of the const, also to be precise, the Partial is not required, but it is more precise since there is a missing key ("tivs")